allcontainers.js

```diff
@@ -98,7 +105,17 @@ function allContainers (opts) {
         return result.emit('error', err)
       }
 
-      containers.forEach(emit)
+      var activeContainers = [];
+      function findPausedContainers(container) {
+        if (container.Status.indexOf("Paused") !== -1) {
+          pausedPreheatContainers[container.Id] = container;
+        } else {
+          activeContainers.push(container);
+        }
+      }
```

Comment:
Just another thought: if we move the definition of this function further away from its one and only use (with exactly one execution), it seems a little bit harder to understand and reason about. I see the code is using a few anonymous functions already, including the call to filter above -- does this really call for separating it out?